### PR TITLE
Fix to_cpu on a multi gpu environment

### PR DESCRIPTION
**Pull request by okuta**
Please merge #1128 before this PR.

### CODE REVIEW

**Comment by okuta** [comment]
See also  #418.

**Comment by unnonouno** [comment]
I merged #1128. Please rebase this branch.

**Comment by okuta** [comment]
@unnonouno I rebased.

File: chainer/cuda.py [hunk 1]
```diff
@@ -215,7 +215,8 @@ def to_cpu(array, stream=None):
 
     """
     if isinstance(array, ndarray):
-        return array.get(stream)
+        with array.device:
```
Comment (unnonouno) [inline chainer/cuda.py]: `with get_device(array):` ?

**Comment by unnonouno** [comment]
LGTM

**Pull request merged**